tests/staticfiles_tests/test_storage.py

```diff
@@ -172,6 +172,7 @@ def test_post_processing(self):
         self.assertIn(os.path.join('cached', 'css', 'window.css'), stats['post_processed'])
         self.assertIn(os.path.join('cached', 'css', 'img', 'window.png'), stats['unmodified'])
         self.assertIn(os.path.join('test', 'nonascii.css'), stats['post_processed'])
+        self.assertIn(os.path.join('cached', 'css', 'ignored.css'), stats['post_processed'])
```

Comment:
Isn't the regression test adding a URL with `chrome:` in `ignored.css`? `test_path_ignored_completely` is already testing this file.